File: A1/A1-036/A1-036.py
# --------------------------------------------------
# File Name : A1-036.py
# Problem   : Squared Sum
# Author    : Worralop Srichainont
# Date      : 2026-03-13
# --------------------------------------------------

# Input
num = int(input())

# Floor to nearest tens
num = (num // 10) * 10

# Output
for i in range(num, -1, -10):
    print(i, end=" ")

File: A1/A1-036/A1-036.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int num;
    cin >> num;
    // Output
    for(int i = num - (num % 10); i >= 0; i -= 10)
    {
        cout << i << ' ';
    }
    return 0;
}